Sandbox/src/SandboxApp.cpp

```diff
@@ -139,19 +139,19 @@ class ExampleLayer : public Hazel::Layer
 
 	void OnUpdate(Hazel::Timestep ts) override
 	{
-		if (Hazel::Input::IsKeyPressed(HZ_KEY_LEFT))
+		if (Hazel::Input::IsKeyPressed(KeyCode::KEY_LEFT))
```

Comment:
KeyCodes and MouseButtonCodes should be moved into the Hazel namespace like all Hazel stuff.